src/app/components/NavBar.tsx

```diff
@@ -33,7 +33,7 @@ const NavItems: { path: string; title: string }[] = [
 
 export function Navbar() {
   return (
-    <nav className="flex items-center justify-between w-full !h-[74px] !min-h-[74px]">
+    <nav className="sticky top-0 sm:static z-50 bg-base-300 flex items-center justify-between w-full !h-[74px] !min-h-[74px]">
```

Comment:
I believe the color should be base-200 (grey) not base-300 (black)
```suggestion
    <nav className="sticky top-0 sm:static z-50 bg-base-200 flex items-center justify-between w-full !h-[74px] !min-h-[74px]">
```